RELEASE.md

```diff
@@ -0,0 +1,114 @@
+# Release Process
+
+This document describes the general process for creating a new release of a
+Marlowe component.
+
+## Step 1: Increment Version Numbers
+
+There are two processes to follow depending on what type of release the current
+release (i.e. the most recent tag on `main`) is: a prerelease or a normal release.
+
+### Process if the current release is a normal release
+
+Increment the version numbers in `marlowe-chain-sync.cabal`, `marlowe-runtime.cabal`,
+`marlowe-runtime-web.cabal`, `marlowe-client.cabal`, and `marlowe-runtime-cli.cabal`
+as prescribed by semantic versioning. Given a version number scheme `major.minor.patch`:
```

Comment:
We have other `.cabal` files, so this number process should include those, too?

Reply:
No - Those other packages are not part of the runtime. I should clarify that this is the set of packages to increment for a runtime release. This document has some strong evidence that the runtime should be a separate repo I think.